lib/logster/redis_store.rb

```diff
@@ -319,5 +409,15 @@ def protected_key
     def grouping_key
       @grouping_key ||= "__LOGSTER__GMAP"
     end
+
+    private
+
+    def register_rate_limit(severities, limit, duration, block)
```

Comment:
its not technically a block ... more like a proc or lambda ... if it was a block it would be &blk and yield